app/models/batch_context.rb

```diff
@@ -177,12 +181,6 @@ def verify_output_dir_no_exists
     # However, these are validated separately, so OK to ignore here.
   end
 
-  def verify_file_manifest_selected_for_media
-    return unless content_structure == 'media' && !using_file_manifest
-
-    errors.add(:content_structure, 'requires a file manifest.  Please indicate you have a file manifest and ensure a file manifest is present.')
-  end
-
```

Comment:
validation for this no longer required since it will always be true now (we still check to be sure the file_manifest.csv actually exists, and that was happening before as well)